packages/account/src/account.ts

```diff
@@ -526,6 +527,50 @@ export class Account {
     return decorated
   }
 
+  buildOnchainSignature(digest: ethers.BytesLike): { bundle: commons.transaction.TransactionBundle, signature: string } {
+    const subdigest = commons.signature.subdigestOf({ digest: ethers.utils.hexlify(digest), chainId: 0, address: this.address })
```

Comment:
There is really no need for it, since the signature will only be valid on the network on which the transaction is sent. Making it more restrictive would only add points of failure.